tests/test_cli.py

```diff
@@ -22,8 +22,10 @@ def test_list_plugins(cmd):
 
 
 @expensive_tests
-@pytest.mark.parametrize("sut", ["DemoMultipleChoiceSUT"])
-@pytest.mark.parametrize("test", ["demo_01", "demo_02", "demo_03"])
+@pytest.mark.parametrize("sut", ["DemoMultipleChoiceSUT", "gpt2", "gpt-3.5-turbo"])
+@pytest.mark.parametrize(
+    "test", ["demo_01", "demo_02", "demo_03", "simple_safety_tests"]
```

Comment:
> This seems solid to me, except that I'd like to have at least some automated test coverage. Perhaps this is a good use for @mkly's daily smoke tests?

I noticed we added `gpt-3.5-turbo`. We won't be able to run that in the daily tests because we do not have an API key for OpenAI in the CI environment. The same appears to be true of llamaguard which requires an API key for together. Not sure what type of challenges we'll face trying to get those added to CI.

Reply:
Before I've happily put secrets in CI, and there are [decent docs](https://docs.github.com/en/actions/security-guides/using-secrets-in-github-actions). My main thought is that it would be nice to have those secrets billed separately, so that we can a) keep an eye on how much CI is costing us, and b) have a better chance of noticing if the CI secrets escape into the wild.

Reply:
If a smoke test isn't the easiest thing, then perhaps you can make some tests along the lines you described in the call? Which I think was running a few examples, checking the output isn't mangled, and making sure some plausible score numbers come out.

Reply:
Done!